.github/workflows/ci.yml

```diff
@@ -179,16 +172,17 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ember-try-scenario: [
-          ember-lts-3.16,
-          ember-lts-3.20,
-          ember-release,
-          ember-beta,
-          ember-default-with-jquery,
-          ember-classic,
-          embroider-safe,
-          embroider-optimized
-        ]
+        ember-try-scenario:
+          [
+            ember-lts-3.28,
+            ember-lts-4.4,
+            ember-lts-4.8,
+            ember-release,
+            ember-beta,
+            ember-default-with-jquery,
```

Comment:
I wonder if we could remove this one too

Reply:
Removed ✅